tethys_apps/cli/test_command.py

```diff
@@ -59,15 +85,34 @@ def test_command(args):
         index_fname = 'index.html'
 
         if args.file and (app_package_tag in args.file or extension_package_tag in args.file):
-            run_process(['coverage', 'html', '--directory={0}'.format(os.path.join(tests_path, report_dirname))])
+            run_process(
+                ['coverage', 'html', '--directory={0}'.format(os.path.join(tests_path, report_dirname))])
         else:
             run_process(['coverage', 'html', config_opt])
 
         try:
-            status = run_process(['open', os.path.join(tests_path, report_dirname, index_fname)])
+            status = run_process(['open', os.path.join(
+                tests_path, report_dirname, index_fname)])
             if status != 0:
                 raise Exception
         except Exception:
-            webbrowser.open_new_tab(os.path.join(tests_path, report_dirname, index_fname))
+            webbrowser.open_new_tab(os.path.join(
+                tests_path, report_dirname, index_fname))
+
+    # Removing Test App
```

Comment:
Is there a reason this is commented out? Seems like a good idea.

Reply:
Just while i was testing repeatedly.